app/models/preserved_object.rb

```diff
@@ -19,8 +19,8 @@ class PreservedObject < ApplicationRecord
 
   # Spawn asynchronous checks of all existing archive complete_moabs.
```

Comment:
not in the scope of this PR, but now that i look at this comment overall, this could probably use fixing too (maybe `Spawn asynchronous checks of each associated ZippedMoabVersion`